lib/cppcheck.cpp

```diff
@@ -413,6 +413,14 @@ static bool reportClangErrors(std::istream &is, const std::function<void(const E
     return false;
 }
 
+std::string CppCheck::getLibraryDumpData() const {
+    std::string out;
+    for (const std::string &s : mSettings.libraries) {
+        out += "  <library lib=\"" +  s + "\"/>\n";
```

Comment:
it seems to me there are 2 spaces between `+  s` not sure why the formatter accepts that.